code/tests/test_BatchPushResults.py

```diff
@@ -15,17 +16,19 @@
 
 @pytest.fixture(autouse=True)
 def get_processor_handler_mock():
-    with patch("backend.batch.BatchPushResults.EmbedderFactory.create") as mock:
-        processor_handler = mock.return_value
-        yield processor_handler
+    with patch("backend.batch.BatchPushResults.EmbedderFactory.create") as mock_create_embedder, \
```

Comment:
Could you get pre-commit up and running so it formats the modified files correctly?

Reply:
@cecheta - this has ran through pre-commit.

The only files to fail are outside the diff of this PR:
![image](https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/assets/2767292/b9afa814-4fe9-4afc-8af0-b8d6900fdfff)

What format is this missing?

Reply:
We don't usually use `\` for new lines, but if `black` is happy with it then that's fine, sorry